lists/TN-RNE

```diff
@@ -0,0 +1,52 @@
+{
+  "name": {
+    "en": "The National Register of Companies",
+    "local": "Registrie National Des Enterprises"
+  },
+  "url": "https://www.registre-entreprises.tn/rne-public/#/recherche-pm",
+  "description": {
+    "en": "The National Register of Companies - RNE- is a public database to collect data and information relating to companies and their availability to the public as well as to State institutions. This is where companies (legal and natural persons) are registered as well as their amending entries.\n"
+  },
+  "coverage": [
+    "TN"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "company"
+  ],
+  "sector": [],
+  "code": "",
+  "confirmed": true,
+  "deprecated": false,
+  "listType": "primary",
+  "access": {
+    "availableOnline": false,
+    "onlineAccessDetails": null,
```

Comment:
For onlineAccessDetails you could add something like "Individual records can be found via the search form. For bulk downloads, there's a fee to pay."